### PR TITLE
Fix German localization

### PR DESCRIPTION
**Pull request by mwallner**
localization working now, cleaned history

### CODE REVIEW

**Comment by RichiCoder1** [comment]
Whoooop. Will review this asap :)

File: Source/ChocolateyGui/ChocolateyGui.csproj [hunk 1]
```diff
@@ -18,6 +18,21 @@
...
+    <UpdateInterval>7</UpdateInterval>
+    <UpdateIntervalUnits>Days</UpdateIntervalUnits>
+    <UpdatePeriodically>false</UpdatePeriodically>
+    <UpdateRequired>false</UpdateRequired>
+    <MapFileExtensions>true</MapFileExtensions>
+    <ApplicationRevision>0</ApplicationRevision>
+    <ApplicationVersion>1.0.0.%2a</ApplicationVersion>
+    <IsWebBootstrapper>false</IsWebBootstrapper>
+    <UseApplicationTrust>false</UseApplicationTrust>
+    <BootstrapperEnabled>true</BootstrapperEnabled>
```
Comment (fwinkelbauer) [inline Source/ChocolateyGui/ChocolateyGui.csproj]: @mwallner if you're wondering why that stuff got added - thats the basic ClickOnce configuration that gets added by Visual Studio if you fool around in the project properties. You could remove it if you're picky

Comment (mwallner) [inline Source/ChocolateyGui/ChocolateyGui.csproj]: @fwinkelbauer seems legit.

File: Source/ChocolateyGui/ChocolateyGui.csproj [hunk 2]
```diff
@@ -18,6 +18,17 @@
...
+    <Install>true</Install>
+    <InstallFrom>Disk</InstallFrom>
+    <UpdateEnabled>false</UpdateEnabled>
+    <UpdateMode>Foreground</UpdateMode>
+    <UpdateInterval>7</UpdateInterval>
+    <UpdateIntervalUnits>Days</UpdateIntervalUnits>
+    <UpdatePeriodically>false</UpdatePeriodically>
+    <UpdateRequired>false</UpdateRequired>
+    <MapFileExtensions>true</MapFileExtensions>
+    <ApplicationRevision>0</ApplicationRevision>
```
Comment (fwinkelbauer) [inline Source/ChocolateyGui/ChocolateyGui.csproj]: Line 21-31 is what you wanted to remove as well ^^

Comment (mwallner) [inline Source/ChocolateyGui/ChocolateyGui.csproj]: :D lol, jup, thanks

**Comment by mwallner** [comment]
@RichiCoder1 please rebase before merge 😇

File: Source/ChocolateyGui/ChocolateyGui.csproj [hunk 3]
```diff
@@ -558,6 +558,18 @@
     <Analyzer Include="..\packages\StyleCop.Analyzers.1.0.0\analyzers\dotnet\cs\StyleCop.Analyzers.CodeFixes.dll" />
     <Analyzer Include="..\packages\StyleCop.Analyzers.1.0.0\analyzers\dotnet\cs\StyleCop.Analyzers.dll" />
   </ItemGroup>
+  <ItemGroup>
```
Comment (RichiCoder1) [inline Source/ChocolateyGui/ChocolateyGui.csproj]: I'm about 90% sure this is also clickonce detrius :)

File: Source/ChocolateyGui/Views/PackageView.xaml [hunk 4]
```diff
@@ -87,23 +87,23 @@
...
+																<TextBlock Text="{x:Static lang:Resources.PackageView_ButtonReinstall}" FontSize="16"/>
                             </StackPanel>
                         </Button>
                         <Button Padding="10" Margin="5 0"
                             Command="{commands:DataContextCommandAdapter Uninstall}">
                             <StackPanel Orientation="Horizontal">
                                 <Image Style="{StaticResource UacActionStyle}" />
                                 <iconPacks:PackIconEntypo Kind="Uninstall" Margin="0 0 5 0 " VerticalAlignment="Center" />
-                                <TextBlock Text="{x:Static lang:Strings.PackageView_ButtonUninstall}" FontSize="16"/>
+																<TextBlock Text="{x:Static lang:Resources.PackageView_ButtonUninstall}" FontSize="16"/>
```
Comment (RichiCoder1) [inline Source/ChocolateyGui/Views/PackageView.xaml]: No a big issue, but any idea what's going on with this spacing?

**Comment by mwallner** [comment]
ok @RichiCoder1 - I srsly don't know what's going on with the spaces - everything looks fine in VS and git bash .. but not on github :disappointed: maybe because I'm a "tab-person" ?

**Comment by RichiCoder1** [comment]
At some point I think we should add a code formatter to take care of those details. I'll fiddle with it later. Lgtm.

**Pull request merged**